distribution/openrct2.d.ts

```diff
@@ -1064,6 +1064,21 @@ declare global {
          * The percentage of downtime for this ride from 0 to 100.
          */
         readonly downtime: number;
+
+        /**
+         * The currently set chain lift speed.
+         */
+        lift_hill_speed: number;
```

Comment:
Please change this to use camel Case for variables. This also applies to the rest.